index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Anja Schmidt's Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="mobile_container">
+      <!-- Nav Bar -->
+      <nav>
+        <ul>
+          <li>Welcome</li>
+          <li><img class="menu" src="images/menu.png" alt="Menu" /></li>
+        </ul>
+      </nav>
+
+      <!-- Header -->
+      <header>
+        <section class="header_container">
+          <h1 class="heading1">Hey there. I’m Anja.</h1>
+          <h1 class="heading2">I’m a software developer</h1>
```

Comment:
- [ ] This should be a `<p>` tag, please kindly fix that as it makes your html more semantic.